yoti-stub/src/services/YotiRequestProcessor.ts

```diff
@@ -739,6 +747,33 @@ export class YotiRequestProcessor {
                         console.log('modifiedPayload', JSON.stringify(modifiedPayload));
                         return new Response(HttpCodesEnum.OK, JSON.stringify(modifiedPayload));
 
+                    case '0150': // UK Passport Success - Only FullName in DocumentFields 
```

Comment:
is it worth adding scenarios for? 

Wrong split of Given Names
Wrong split of Surnames
Incorrect sequence of names
Name with non space characters returned differently

https://docs.google.com/spreadsheets/d/1ybKJNj88zVUdg3Pl44V3hQpcvy9Ws6w9VJ8iKS9wmSU/edit?pli=1#gid=0

These should issue a VC but fail at SPOT(Staging)

Reply:
These look like they are existing scenarios as they are listed as "no change". @Z-Mohamoud do we want to add these extra regression tests?